pkgs/development/python-modules/click/default.nix

```diff
@@ -9,6 +9,13 @@ buildPythonPackage rec {
     sha256 = "02qkfpykbq35id8glfgwc38yc430427yd05z1wc5cnld8zgicmgi";
   };
 
+  patches = [
+    (substituteAll {
+      src = ./fix-paths.patch;
+      locale = "${glibc.bin}/bin/locale";
```

Comment:
Should be only applied on linux.
btw. @dtzWill how does musl handle locales?

Reply:
I think we could probably add this to unix-tools.nix. Right now I don't think Musl comes with a locale implementation but NetBSD has a portable version.

Reply:
http://pubs.opengroup.org/onlinepubs/9699919799/utilities/locale.html

Reply:
so if you can use "unixtools.locale" here that would be more portable. we will eventually add more implementations - right now it is just the glibc one.

Reply:
Apparently there's this: https://gitlab.com/rilian-la-te/musl-locales which I've packaged here: https://github.com/dtzWill/nixpkgs/tree/feature/musl-locales

The tool maybe kinda works--IIRC there's some environment variable to set that tells musl where to look? Honestly not sure :).

Reply:
How about os x in that case? Would the netbsd version work there?

Reply:
Maybe... i think apple also has its own version

Reply:
Is this part of `darwin.system_cmds`?

Reply:
> so if you can use "unixtools.locale" here that would be more portable. we will eventually add more implementations - right now it is just the glibc one.

@matthewbauer `unixtools.locale` doesn't seem to exist currently, or am I'm missing something obvious? ;-)

> The tool maybe kinda works--IIRC there's some environment variable to set that tells musl where to look? Honestly not sure :).

@dtzWill looks like that might be [`MUSL_LOCPATH` for `musl-locales`](https://gitlab.com/rilian-la-te/musl-locales/blob/master/locale.c#L107), similar to `LOCALE_ARCHIVE` for `glibc`. But I'm not sure if it's the same when running applications linked against musl…